tests/services/test_schedule_service.py

```
@@ -126,3 +128,35 @@ async def test_dose_intervals(
+    # invalid timezone string → pytz.UnknownTimeZoneError
+    bad_user = User(id=99, timezone="Invalid/Zone")
+    with patch("src.services.schedule_service.datetime", wraps=datetime) as mock_dt:
+        mock_dt.now.return_value = datetime(2024, 1, 1, 7, tzinfo=timezone.utc)
+        with pytest.raises(pytz.UnknownTimeZoneError):
+            await service.get_next_dose_time(bad_user, schedule)
+
+
+@pytest.mark.asyncio
+async def test_create_schedule_error(service, schedule):
+    service.session.commit.side_effect = Exception("Create failed")
+    with pytest.raises(Exception):
```

Ruff (0.11.9)

Do not assert blind exception: `Exception`

(B017)